standup/script.js

```diff
@@ -121,8 +121,21 @@ function createTableHeaderElement() {
     type: 'th',
     classList: ['user', 'date', 'table-head'],
   });
-  headerCellElement.innerHTML = 'DATES ➡️<hr />USERS ⬇️';
+  headerCellElement.innerHTML =
+    'DATES➡️ <br><br><input type="date" id="date" name="DATES" max=""/> <hr />USERS ⬇️';
```

Comment:
- let's create another element for the input and add it to header cell element instead of directly adding the html as a string here and using <br /> tag

Reply:
Made the requested changes